jquery.flot.js

```diff
@@ -1490,7 +1490,7 @@ Licensed under the MIT license.
         function defaultTickFormatter(value, axis, precision) {
             var oldTickDecimals = axis.tickDecimals;
 
-            if (precision) {
+            if (precision && precision > 0) {
```

Comment:
I can swear I've seen this check being added in the past. Could you please check there are no other spots left?